sched/signal/sig_default.c

```diff
@@ -194,11 +194,20 @@ static void nxsig_abnormal_termination(int signo)
 {
   FAR struct tcb_s *rtcb = (FAR struct tcb_s *)this_task();
 
-  /* Notify the target if the non-cancelable or deferred cancellation set */
+#ifdef CONFIG_CANCELLATION_POINTS
```

Comment:
Should we skip the cancellation point check here? It doesn't make sense that the default handle do the check, but user installed handler doesn't.

Reply:
Done